src/analyses/goto_check.h

```diff
@@ -26,6 +26,7 @@ void goto_check(
 void goto_check(
   const namespacet &ns,
   const optionst &options,
+  const irep_idt &mode,
```

Comment:
A string called "mode" is a bit leading ... no chance of a doxygen comment on what the inputs are is there?